src/app/header/header.component.html

```diff
@@ -5,8 +5,7 @@
           <div class='col-md-3'>
             <span>Computer Management System</span>
           </div>
-          <div class='col-md-3'></div>
-          <div class='col-md-3'></div>
+          <div class='col-md-6'></div>
```

Comment:
Not yet ;) https://getbootstrap.com/docs/4.0/layout/grid/#offset-classes
Add offset-md-6 to the div with content